core/src/main/scripts/importer/validateData.py

```diff
@@ -48,7 +48,7 @@
 from abc import ABCMeta, abstractmethod
 
 # configure relative imports if running as a script; see PEP 366
-if __name__ == "__main__" and __package__ is None:
+if __name__ == "__main__" and (__package__ is None or __package__ is '') :
```

Comment:
This is the only way I can get this to work in VScode. Here __package__ returns '' not None. May this be caused by a different python interpreter (I use 3.6.7)?

Reply:
@fedde-s any ideas?

Reply:
This change does not belong in the code, it's an undocumented workaround for a misconfigured IDE.

As the PEP referenced on the line above says, `__package__` will be `None` if the file is run as a script, or it will be `importer` if the file is run from the module search path as `importer.validateData` (or as `from importer import validateData`).

Reply:
As a more proper solution, it be worth refactoring the package structure to remove the need for this boilerplate, by either:
- replacing the relative imports by absolute ones (as the folder containing the script [will automatically end up](https://docs.python.org/3/tutorial/modules.html#the-module-search-path) in the search path), or
- renaming `cbioportalImporter.py` and validateData.py` and creating new wrapper scripts in their place, so that only the wrapper scripts need to do an absolute import.

Both options will require a change to the way the automated tests import these scripts, and should probably be reviewed as a separate pull request.

Reply:
I do not see how my IDE is configured for this. I have removed this change.